producer.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3
+# thoth-graph-refresh-job-producer
+# Copyright(C) 2020 Sai Sankar Gochhayat, Fridolin Pokorny
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Refresh data stored in the graph database.
+
+Produces messages regarding package depending on the component in which is used,
+which are picked up by investigator to schedule solvers.
```

Comment:
`Produces messages regarding knowledge missing about a certain package. Those messages are handled by thoth-investigator which decide which workflows need to run to acquire that knowledge.